### PR TITLE
feat/issue-11472: Admin view of Tokens and Airdrops home - pre-Owner token minting

### LINKED ISSUE #11472: Admin view of Tokens and Airdrops home - pre-Owner token minting
A couple of new screens have been added to the Mint Owner token flows to show what an admin sees when navigating to the Tokens and Airdrops home screens prior to the Owner token being minted.

**EXISTING: Community founder view of Tokens and Airdrops home screens:**
The Community founder sees a `Get started` panel which introduces them to the concept of the Owner token, explaining why minting is disabled until the Owner token is minted. It also contains a CTA to `Mint Owner token`

Figma link:
https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=34794-589770&mode=design&t=H9WXPVH15Vuk0mSS-4

Tokens

Airdrop

**NEW: Admin view of Tokens and Airdrops home screens:**
The admin view of these screens (before the Owner token is minted) contains a modified version of the `Get started` panel. The copy is adjusted to introduce the concept of the TokenMaster token to them and to encourage them to reach out to the Community founder if they would like TokenMaster permissions.

Figma link:
https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=34794-589843&mode=design&t=H9WXPVH15Vuk0mSS-4

Tokens

Airdrop

Once the Community Owner and TokenMaster tokens have been minted, the Owner `Get started` panel is removed from the UI. However, the admin view of the `Get started` panel should persist even after the Owner token is minted given it is still relevant.

cc @noeliaSD

### PR DESCRIPTION
**Pull request by noeliaSD**
Closes #11472

### What does the PR do

- New `Get started` panel for admins in `Tokens` and `Airdrops` section added.
- Added support in `storybook`.

### Affected areas

Community Settings / Tokens and Airdops

### Screenshot of functionality 
- Tokens:

https://github.com/status-im/status-desktop/assets/97019400/39954171-3d4c-4998-84dc-6fb6c7f66fc3

- Airdrops:

https://github.com/status-im/status-desktop/assets/97019400/6d969ce3-d268-4747-8bbd-53f6df83ac7c

### CODE REVIEW

File: ui/app/AppLayouts/Communities/panels/AirdropsSettingsPanel.qml [hunk 1]
```diff
@@ -14,6 +14,8 @@ StackView {
     // id, name, image, color, owner properties expected
     required property var communityDetails
     required property bool isOwner
+    required property bool isTokenMasterOwner
```
Comment (caybro) [inline ui/app/AppLayouts/Communities/panels/AirdropsSettingsPanel.qml:17]: just `isTokenMaster`?

File: ui/app/AppLayouts/Communities/panels/AirdropsSettingsPanel.qml [hunk 2]
```diff
@@ -53,6 +55,8 @@ StackView {
     QtObject {
         id: d
 
+        property bool isOnlyAdmin: root.isAdmin && !root.isOwner && !root.isTokenMasterOwner
```
Comment (caybro) [inline ui/app/AppLayouts/Communities/panels/AirdropsSettingsPanel.qml:58]: `readonly`

Comment (caybro) [reply ui/app/AppLayouts/Communities/panels/AirdropsSettingsPanel.qml:58]: And, I'd rename this to `isAdminOnly`

`isOnlyAdmin` means "I am the only admin here", and not "I am just an admin, and not an owner or token master"

File: ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml [hunk 3]
```diff
@@ -99,7 +99,7 @@ StackView {
         title: qsTr("Tokens")
 
         buttons: DisabledTooltipButton {
-            property bool onlyAdmin: root.isAdmin && !root.isOwner
+            property bool onlyAdmin: root.isAdmin && !root.isOwner && !root.isTokenMasterOwner
             property bool buttonEnabled: (root.isOwner || root.isTokenMasterOwner) && d.isTokenOwnerDeployed
```
Comment (caybro) [inline ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml:103]: `readonly` both, and `adminOnly`

File: ui/app/AppLayouts/Communities/views/MintedTokensView.qml [hunk 4]
```diff
@@ -99,13 +100,17 @@ StatusScrollView {
             }
 
             StatusInfoBoxPanel {
+
+                property bool isOnlyAdmin: root.isAdmin && !root.isOwner
```
Comment (caybro) [inline ui/app/AppLayouts/Communities/views/MintedTokensView.qml:104]: `readonly` + `isAdminOnly`

**Review by caybro — COMMENTED** [review]
Looks cool, just some wording suggestions

File: ui/app/AppLayouts/Communities/panels/AirdropsSettingsPanel.qml (hunk 1 above)
Comment (noeliaSD) [reply ui/app/AppLayouts/Communities/panels/AirdropsSettingsPanel.qml:17]: I would like to leave it as `isTokenMasterOwner` bc there are 2 concepts here that could be mixed up:
- First one is the concept of having the `token master token` deployed (that maybe we will need to add in some point a property like `isTokenMaster` or `isTokenMasterDeployed` or `tokenMasterExists`.
- Second one is the concept of owning this type of token, so for being more explicit and to not have confusions I've named it like this: `isTokenMasterOwner`.

WDYT?

File: ui/app/AppLayouts/Communities/panels/AirdropsSettingsPanel.qml (hunk 2 above)
Comment (noeliaSD) [reply ui/app/AppLayouts/Communities/panels/AirdropsSettingsPanel.qml:58]: Ok, got it!!! Will update

**Comment by noeliaSD** [comment]
Pushed updates! @caybro please, take a second look!

**Review by caybro — APPROVED** [review]

**Review by micieslak — APPROVED** [review]
Looks very good for me!

**Pull request merged**